di_website/events/models.py

```diff
@@ -28,10 +28,7 @@ class EventPage(TypesetBodyMixin, HeroMixin, Page):
 
     start_date = models.DateField(default=datetime.now)
     end_date = models.DateField(default=datetime.now)
-    start_time = models.TimeField(blank=True, null=True)
-    end_time = models.TimeField(blank=True, null=True)
-    timezone = models.CharField(max_length=160, help_text='Timezone information for event', blank=True, null=True)
-    location = models.CharField(max_length=100, help_text='Physical location of event', blank=True, null=True)
+    time_and_location = models.TextField(max_length=160, help_text='Time and location information for event', blank=True, null=True)
```

Comment:
![image](https://user-images.githubusercontent.com/5672438/97995820-1537b780-1df8-11eb-882e-3156dceb8f1b.png)

I don't think it's that much work to create two free text fields, one for time & one for location as requested in the text above ... that way we can at least preserve the icons. The location one was already there, so it's just a matter of a free-text time field. Would also cut down on your data migration hustles.

Reply:
Updated with two fields for time and location